pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -301,6 +301,103 @@ type MonitoringSpec struct {
 	// +kubebuilder:validation:Maximum=65535
 	// +optional
 	ExternalMgrPrometheusPort uint16 `json:"externalMgrPrometheusPort,omitempty"`
+	// CustomizedPrometheus points to a customized Ceph prometheus alerts
+	// +optional
+	// +nullable
+	CustomizedPrometheus *PrometheusRuleCustomized `json:"prometheusRule,omitempty"`
```

Comment:
What about this name to clarify that there are defaults and these are just overriding the defaults?
```suggestion
	CustomizedPrometheus *PrometheusRuleCustomized `json:"prometheusRuleOverrides,omitempty"`
```

Reply:
Or perhaps we just pull the alert overrides in directly here. If other overrides are needed than alerts, it can just be a setting at this level as well.
```suggestion
	AlertruleOverrides *Alerts `json:"alertRuleOverrides,omitempty"`
```

Reply:
If we decide to keep the option to add other sections (for example, "label section"), I think we should go with your first suggestion.
What do you think?

Reply:
If the labels need to be customizable, wouldn't the `labels` field just be added on the `CephAlert` struct? Seems like it's just part of that struct instead of creating something separate. If so, I would recommend the latter option.

Reply:
I understand, I changed to the latter option